src/beyond_bets/transforms/market_conditions.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime, timedelta
+from pyspark.sql import DataFrame, SparkSession
+from beyond_bets.base.transform import Transform
+from beyond_bets.datasets.bets import Bets
+from pyspark.sql import functions as fn
+
+
+spark = SparkSession.builder.getOrCreate()
+
+
+class BetGrader(Transform):
+    def __init__(self, start_time: datetime):
```

Comment:
Had to introduce a start time due to the data generated. It creates data backwards from 00:00:00 of today, therefore cannot test for the last 15 mins as there will be no data points available to test for now.